.github/workflows/acceptance.yml

```diff
@@ -45,6 +45,9 @@ jobs:
     name: acceptance
     runs-on: ubuntu-22.04
     timeout-minutes: 30
+    permissions:
+      contents: read
```

Comment:
```suggestion
```
By default, permission to read contents is granted to the token, so no need to specify them.
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
It doesn't hurt to be explicit